### PR TITLE
Added new character "alien"

### PR DESCRIPTION
**Pull request by toxicdeath32**
Hi, I've added a new character, but I am currently unable to claim that task right now as I am still waiting for approval on another one, I will claim it once ready. Do advice on how to improve on the design if possible.

### CODE REVIEW

File: javascript/flapping.js [hunk 1]
```diff
@@ -82,6 +82,7 @@ var flappy = null;
 function characterChange(layer) {
     hide_layer('bird');
     hide_layer('bat');
+	hide_layer('alien');
```
Comment (niccokunzmann) [inline javascript/flapping.js]: please be consistent and use spaces.

File: flappy.svg [hunk 2]
```diff
@@ -9321,12 +9405,12 @@
...
-         x="205.98065"
-         y="191.03279"
+         x="255.98065"
+         y="203.03279"
          style="text-align:center;text-anchor:middle">JoJo Let's </tspan><tspan
          sodipodi:role="line"
-         x="223.67596"
-         y="255.53278"
+         x="243.67596"
+         y="290.53278"
```
Comment (niccokunzmann) [inline flappy.svg]: If you move the welcome text, maybe it should not be there in the first place. Either make it readable or remove it.

File: flappy.svg [hunk 3]
```diff
@@ -9397,7 +9481,7 @@
         <path
            inkscape:connector-curvature="0"
            class="octo-arm"
-           style="fill:#ff6600;transform-origin:130px 106px"
```
Comment (niccokunzmann) [inline flappy.svg]: This line is important for the octopusarm in the top right corner. feel free to put `transform-origin:130px 106px` into the css file.

**Comment by niccokunzmann** [comment]
I like the character! It is funny :) 
Making your character this wide, makes it harder to play it because reaction times are lower. Just so you know. It is good to have characters with different difficulties.

**Comment by PohSayKeong** [comment]
I have fixed the problems as you said. :)

**Comment by PohSayKeong** [comment]
Hi, I've fixed the problems here. The one working on this task is actually my friend and he was using my account to make the commit and I forgot to change the git configuration...

**Comment by niccokunzmann** [comment]
I am reviewing...

**Comment by niccokunzmann** [comment]
It looks good. Thanks for adding the character. And... We have an App in the Google Playstore now, which should include your character, too :)

**Comment by niccokunzmann** [comment]
You should add your name to the contributors file in the master branch.

**Pull request merged**